Resources/Maps/lighthouse.yml

```
@@ -95024,26 +95024,26 @@ entities:
       rot: -1.5707963267948966 rad
       pos: 20.5,-64.5
       parent: 2
-- proto: HighSecDoor
+- proto: HighSecEngineeringLocked
   entities:
   - uid: 13819
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)